### PR TITLE
UITEN-156 Bursar exports configuration form

### PR DESCRIPTION
**Pull request by NikitaSedyx**
## Purpose

https://issues.folio.org/browse/UITEN-156

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater (covered by Jess - sonar uses bigtest...)
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Review by aliaksei-chumakou — APPROVED** [review]

File: src/settings/BursarExports/BursarExportsConfiguration/BursarExportsConfiguration.js [hunk 1]
```diff
@@ -0,0 +1,188 @@
+import React, { useEffect, useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { Field } from 'react-final-form';
```
Comment (axelhunn) [inline src/settings/BursarExports/BursarExportsConfiguration/BursarExportsConfiguration.js:3]: my recommendations to useField

File: src/settings/BursarExports/BursarExportsConfiguration/BursarExportsConfiguration.test.js [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+  it('should render schedule period field', () => {
+    const { getByText } = renderBursarExportsConfiguration();
+
+    expect(getByText('ui-tenant-settings.settings.bursarExports.schedulePeriod')).toBeDefined();
+  });
+
+  it('should render ftp address field', () => {
+    const { getByText } = renderBursarExportsConfiguration();
+
+    expect(getByText('ui-tenant-settings.settings.bursarExports.ftpAddress')).toBeDefined();
```
Comment (axelhunn) [inline src/settings/BursarExports/BursarExportsConfiguration/BursarExportsConfiguration.test.js:31]:
```suggestion
    expect(getByLabelText('ui-tenant-settings.settings.bursarExports.ftpAddress')).toBeVisible();
```

**Review by axelhunn — APPROVED** [review]

**Review by ViktorSoroka07 — APPROVED** [review]

File: src/settings/BursarExports/BursarExports.js [hunk 3]
```diff
@@ -23,11 +41,10 @@ export const BursarExports = () => {
...
       paneTitle={formatMessage({ id: 'ui-tenant-settings.settings.bursarExports' })}
     >
-      <Row>
-        <Col>
-          Bursar exports configuration
-        </Col>
-      </Row>
+      <BursarExportsConfiguration
+        onFormStateChanged={setBursarConfigForm}
+        onSubmit={(values) => { console.log(values); }}
```
[the remark this answers is not in the document]
Comment (NikitaSedyx) [reply src/settings/BursarExports/BursarExports.js:46]: yes, will be replaced in another story, BE is not ready

File: src/settings/BursarExports/BursarExportsConfiguration/BursarExportsConfiguration.js (hunk 1 above)
Comment (NikitaSedyx) [reply src/settings/BursarExports/BursarExportsConfiguration/BursarExportsConfiguration.js:3]: imo Field is more readable + there are conditional fields here

File: src/settings/BursarExports/BursarExportsConfiguration/BursarExportsConfiguration.test.js (hunk 2 above)
Comment (NikitaSedyx) [reply src/settings/BursarExports/BursarExportsConfiguration/BursarExportsConfiguration.test.js:31]: `expect(...).toBeVisisble is not a function` + `getByLabelText` can't find label - no idea why

Comment (axelhunn) [reply src/settings/BursarExports/BursarExportsConfiguration/BursarExportsConfiguration.test.js:31]: `import '@testing-library/jest-dom'` for `toBeVisisble`

**Pull request merged**